Album/ViewControllers/MapControllers/FriendMapViewController.m

```diff
@@ -154,33 +160,33 @@ - (UIView*) mapView:(GMSMapView *)mapView markerInfoWindow:(nonnull GMSMarker *)
     // Array of pins from the specific coordinate
     NSArray *pinsFromCoord = [self fetchPinsFromCoord:coordinate];
     // Check if exisitng pins exist from this coordinate
-    if(pinsFromCoord && pinsFromCoord.count > 0) {
+    if (pinsFromCoord && pinsFromCoord.count > 0) {
         PFObject *firstPin = pinsFromCoord[0];
-        int i=0;
-        while(i<pinsFromCoord.count) {
+        int i = 0;
+        while (i < pinsFromCoord.count) {
             PFObject *pin = pinsFromCoord[i];
             // Save pins into cache data structure
-            if(!self.placeToPins[pin[@"placeName"]]) {
+            if (!self.placeToPins[pin[@"placeName"]]) {
                 [self.placeToPins setObject:[[NSMutableArray alloc]init] forKey:pin[@"placeName"]];
             }
             [self.placeToPins[pin[@"placeName"]] addObject:pin];
             // Save images of the specific pin to the cache data structure
-            NSArray* imagesFromPin = [self imagesFromPin:pin.objectId];
+            NSArray *imagesFromPin = [self imagesFromPin:pin.objectId];
             [self.pinImages setObject:imagesFromPin forKey:pin.objectId];
             i++;
         }
         InfoMarkerView *markerView = [[[NSBundle mainBundle] loadNibNamed:@"InfoExistWindow" owner:self options:nil] objectAtIndex:0];
         // Set image of the info window to first in the array
-        NSArray* imagesFromPin = self.pinImages[firstPin.objectId];
-        if(imagesFromPin && imagesFromPin.count > 0) {
+        NSArray *imagesFromPin = self.pinImages[firstPin.objectId];
+        if (imagesFromPin && imagesFromPin.count > 0) {
             PFFileObject *imageFile = imagesFromPin[0][@"imageFile"];
             [markerView.pinImageView setFile:imageFile];
             [imageFile getDataInBackgroundWithBlock:^(NSData *imageData, NSError *error) {
-                     if (!error) {
-                     UIImage *image = [UIImage imageWithData:imageData];
-                     [markerView.pinImageView setImage:image];
-                 }
-             }];
+                           if (!error) {
+                           UIImage *image = [UIImage imageWithData:imageData];
+                           [markerView.pinImageView setImage:image];
+                           }
+                       }];
```

Comment:
If there is indeed an error? consider error handling (not necessarily in this PR)

Reply:
Alert added!